### PR TITLE
[add] account_due_list (fixed)

### PR DESCRIPTION
**Pull request by elacunza**
This is a follow-up to #84, fixing the issues highlighted there.

### CODE REVIEW

**Comment by eLBati** [comment]
LGTM :+1:

File: account_due_list/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': "Payments Due list",
+    'version': '0.1',
+    'category': 'Generic Modules/Payment',
+    'description': """
```
Comment (yvaucher) [inline account_due_list/__openerp__.py:31]: In 8.0 description must go in `README.rst` to be visible on github, on apps.odoo.com and in Odoo module description.

Here is the template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by yvaucher** [comment]
If you have no icon for this module here is the generic one for OCA modules: https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.png

File: account_due_list/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+A due list of pending payments. The list contains every expected payment,
+generated by invoices. The list is fully filterable.
+    """,
+    'author': 'Agile Business Group, Zikzakmedia SL',
```
Comment (yvaucher) [inline account_due_list/__openerp__.py]: Please add `Odoo Community Association (OCA)`

**Comment by yvaucher** [comment]
Please fix those little "cosmetics" issues otherwise code LGTM

**Comment by elacunza** [comment]
Ok, let me know if these fixes are OK. Not very sure about the icon thing. Thanks

**Comment by yvaucher** [comment]
For icon you only need  account_due_list/static/description/icon.png 

Nevertheless Icon seems corrupted, I can't open it.

**Comment by elacunza** [comment]
Sorry, I made a mistake downloading the PNG/SVG. Now fixed. Thanks.

**Comment by yvaucher** [comment]
Much better :+1:

**Comment by omar7r** [comment]
:+1: tested

**Comment by gurneyalex** [comment]
```
./account_due_list/__openerp__.py:35:80: E501 line too long (87 > 79 characters)
```

**Comment by elacunza** [comment]
Thanks for pointing it out

**Comment by eLBati** [comment]
Following @pedrobaeza [proposal](https://www.odoo.com/groups/community-59/community-12267143) about keeping https://github.com/OCA/account-payment alive, I propose to move this module to `account-payment` repository, because the module is not related to _bank_ payments only

**Comment by pedrobaeza** [comment]
I agree

**Comment by elacunza** [comment]
I'll set up a new PR against that repo; when one of the PRs gets pulled the other can be closed.

**Comment by elacunza** [comment]
https://github.com/OCA/account-payment/pull/34

**Comment by tafaRU** [comment]
@elacunza, I see another PR about migration of the module ```account_due_list```: this and https://github.com/OCA/account-payment/pull/34 Which one is the right PR?

**Comment by pedrobaeza** [comment]
This one can be closed, as the module has been relocated.

**Pull request closed without merging**